packages/dd-trace/src/opentelemetry/span.js

```diff
@@ -217,6 +218,14 @@ class Span {
     return this
   }
 
+  addSpanPointer (attributes) {
+    const zeroContext = new SpanContext({
+      traceId: id('0'),
+      spanId: id('0')
+    })
+    return this.addLink(zeroContext, attributes)
```

Comment:
arguably we could take care of the link kind stuff here so that callers don't have to know that particular aspect of span pointer storage details. for now they're embedded in the span links list, but in the future we might change that and it would be easier to do so if we didn't leak the span link abstraction out of this function.

Reply:
Agreed, I'm just going to make that change now because this is a good suggestion